src/Compilers/CSharp/Test/Emit/CodeGen/IndexAndRangeTests.cs

```diff
@@ -4236,5 +4214,170 @@ static void M(Span<int> s)
             var comp = CreateCompilation(source, targetFramework: TargetFramework.Net70);
             comp.VerifyDiagnostics();
         }
+
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
+        public void SingleOverloadReadOnlySpan(bool isMissing)
```

Comment:
Please add a version of these tests that uses the Span from corlib as well, so we test both the Special and WellKnown infrastructure.

Reply:
Which should I use to prevent the execution in net472, `ConditionalFact`, `#if NET`, or `ExecutionConditionUtil.IsCoreClr`?

Reply:
Again, you're only skipping the binary execution, not compilation. See other examples of ExecutionConditionUtil for examples.

Reply:
I managed to fulfill your requirements by combining, `targetFramework: TargetFramework.Net70`, `expectedOutput: null`, and `verify: Verification.FailsPEVerify`.